experiments/ClimaEarth/run_amip.jl

```diff
@@ -308,9 +273,14 @@ if sim_mode <: AMIPMode
         comms_ctx,
         date0,
         mono_surface,
-        land_area_fraction,
+        land_fraction,
     )
 
+    ## ocean model using prescribed data
+    ice_fraction = Interfacer.get_field(ice_sim, Val(:area_fraction))
+    ocean_fraction = FT(1) .- ice_fraction .- land_fraction
+    ocean_sim = PrescribedOceanSimulation(FT, boundary_space, date0, t_start, ocean_fraction, thermo_params, comms_ctx)
```

Comment:
The ocean model init is moved after the sea ice model init so we are consistent in our priority of area fractions (land fraction is applied first, then sea ice, then ocean). This now matches the ordering in the FieldExchanger module